buildenv/buildenv.nix

```diff
@@ -68,7 +68,7 @@ let
   environmentOutputs = [
     "runtime"
     "develop"
-  ] ++ (builtins.map (x: "build-${x}") (builtins.attrNames build));
+  ] ++ (builtins.map (buildId: "build-${buildId}") (builtins.attrNames buildSection));
```

Comment:
I thought we used `pname` in virtually every other context for this string - can we not use that here, or is it the case that y'all use `buildId` in the CLI? (Not repeating comment for use of `buildId` below, but the same point holds.)